src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -45,18 +39,13 @@
 
     public static final String VALID_PHONE_AMY = "11111111";
     public static final String VALID_PHONE_BOB = "22222222";
-    public static final String VALID_NAME_AMY = "Amy";
-    public static final String VALID_BOOKING_START1 = "030321";
-    public static final String VALID_BOOKING_START2 = "300321";
-    public static final String VALID_BOOKING_END1 = "050521";
-    public static final String VALID_BOOKING_END2 = "300521";
-
-    public static final String BOOKING_DETAILS_DESC_RESIDENCE1 = " " + PREFIX_BOOKING + VALID_BOOKING_DETAILS;
-    public static final String BOOKING_DETAILS_DESC_RESIDENCE2 = " " + PREFIX_BOOKING + VALID_BOOKING_DETAILS;
+    public static final String VALID_BOOKING_START1 = "03-03-2021";
+    public static final String VALID_BOOKING_START2 = "30-03-2021";
+    public static final String VALID_BOOKING_END1 = "05-05-2021";
+    public static final String VALID_BOOKING_END2 = "30-05-2021";
+
     public static final String NAME_DESC_RESIDENCE1 = " " + PREFIX_NAME + VALID_NAME_RESIDENCE1;
     public static final String NAME_DESC_RESIDENCE2 = " " + PREFIX_NAME + VALID_NAME_RESIDENCE2;
-    public static final String PHONE_DESC_AMY = " " + PREFIX_PHONE + VALID_PHONE_AMY;
-    public static final String PHONE_DESC_BOB = " " + PREFIX_PHONE + VALID_PHONE_BOB;
```

Comment:
I think we need these to make bookings to be used in tests later but maybe could rename to PHONE_DESC_BOOKING1 and PHONE_DESC_BOOKING2?

Reply:
This sounds sensible, I have replaced references to "Amy" and "Bob" with "Booking1" and "Booking2" respectively.